ignite/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: Ignite
+
+## Overview
+
+This check monitors [Ignite][1].
+
+## Setup
+
+### Installation
+
+The Ignite check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
+
+### Configuration
```

Comment:
Check if configuration is actually needed on ignite side

Reply:
Done.